### DISCUSSION TITLE
Will this run on an Onju Voice?

### DISCUSSION THREAD

**Discussion by droyholmes** [sle118/squeezelite-esp32#445] [Q&A]
For those unfamiliar Onju Voice is a PCB that is a direct drop in replacement for Google Nest Mini 2nd Generation. More details [here](https://github.com/justLV/onju-voice)

I'm wondering if anyone has gotten squeezelite-esp32 running on one of these, or if it is even possible?

**Comment by philippe44** [comment]
I'd say probably assuming the s3 has PSRAM.
